src/lib/components/scheduling/SongAvailibilityTimeline.svelte

```diff
@@ -159,6 +161,10 @@
 		background-color: #ef0612;
 	}
```

Comment:
is red still needed?

Reply:
Yes, currently for when people are partly unavailable, that part is red, but I will improve it in a future issue.